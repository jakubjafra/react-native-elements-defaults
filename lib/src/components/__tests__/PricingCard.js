const testHelperUtils = require('../../testHelperUtils');

const utils = require('../../utils');
const Overwrite = utils.Overwrite;

const name = 'PricingCard',
    __oldElement__ = require('react-native-elements/src/pricing/PricingCard'),
    __element__ = require('../PricingCard'),
    noTheme = new Overwrite(
        name,
        Overwrite.TYPE.FUNCTIONAL_COMPONENT_ATTRIBUTES,
        Overwrite.SCOPE.DEFAULTS,
        {}
    ),
    basicTheme = new Overwrite(
        name,
        Overwrite.TYPE.FUNCTIONAL_COMPONENT_ATTRIBUTES,
        Overwrite.SCOPE.DEFAULTS,
        {
            containerStyle: {
                backgroundColor: 'red',
            }
        }
    );

describe(name, () => {
    testHelperUtils.defaultTests(name, __oldElement__, __element__, noTheme, basicTheme);
});
