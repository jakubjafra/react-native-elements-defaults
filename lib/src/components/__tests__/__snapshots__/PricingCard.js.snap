// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PricingCard with basic theme applied OldPricingCard renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "white",
        "borderColor": "#e1e8ee",
        "borderWidth": 1,
        "margin": 15,
        "marginBottom": 15,
        "padding": 15,
        "shadowColor": "rgba(0,0,0, .2)",
        "shadowOffset": Object {
          "height": 1,
          "width": 0,
        },
        "shadowOpacity": 0.5,
        "shadowRadius": 0.5,
      },
      undefined,
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
        },
        undefined,
      ]
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {},
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          Array [
            Object {
              "color": "#9E9E9E",
              "fontSize": 37.5,
              "fontWeight": "800",
              "textAlign": "center",
            },
            Object {
              "color": "#9E9E9E",
            },
            undefined,
          ],
        ]
      }
    />
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {},
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          Array [
            Object {
              "fontSize": 50,
              "fontWeight": "700",
              "marginBottom": 10,
              "marginTop": 10,
              "textAlign": "center",
            },
            undefined,
          ],
        ]
      }
    />
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {},
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          Array [
            Object {
              "color": "#86939e",
              "fontWeight": "600",
              "marginBottom": 5,
              "marginTop": 5,
              "textAlign": "center",
            },
            undefined,
          ],
        ]
      }
    >
      1 User
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {},
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          Array [
            Object {
              "color": "#86939e",
              "fontWeight": "600",
              "marginBottom": 5,
              "marginTop": 5,
              "textAlign": "center",
            },
            undefined,
          ],
        ]
      }
    >
      Basic Support
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {},
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          Array [
            Object {
              "color": "#86939e",
              "fontWeight": "600",
              "marginBottom": 5,
              "marginTop": 5,
              "textAlign": "center",
            },
            undefined,
          ],
        ]
      }
    >
      All Core Features
    </Text>
    <View
      style={
        Array [
          Object {
            "marginLeft": 15,
            "marginRight": 15,
          },
          undefined,
          undefined,
          undefined,
        ]
      }
    >
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        nativeID={undefined}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            undefined,
          ]
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <View
          pointerEvents="box-only"
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "#9E9E9E",
                "flexDirection": "row",
                "justifyContent": "center",
                "padding": 19,
              },
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              Object {
                "padding": 12,
              },
              undefined,
              undefined,
              undefined,
              Array [
                Object {
                  "marginBottom": 10,
                  "marginTop": 15,
                },
                undefined,
                Object {
                  "backgroundColor": "#9E9E9E",
                },
              ],
              undefined,
              undefined,
            ]
          }
        >
          <Text
            accessible={true}
            allowFontScaling={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "white",
                  "fontSize": 18,
                },
                Array [
                  Object {
                    "marginRight": 10,
                  },
                  undefined,
                ],
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
              ]
            }
          >
            î¤…
          </Text>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Array [
                Object {},
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                Array [
                  Object {
                    "color": "white",
                    "fontSize": 20,
                  },
                  undefined,
                  Object {
                    "fontSize": 17.5,
                  },
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                ],
              ]
            }
          >
            GET STARTED
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;

exports[`PricingCard with basic theme applied PricingCard renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "white",
        "borderColor": "#e1e8ee",
        "borderWidth": 1,
        "margin": 15,
        "marginBottom": 15,
        "padding": 15,
        "shadowColor": "rgba(0,0,0, .2)",
        "shadowOffset": Object {
          "height": 1,
          "width": 0,
        },
        "shadowOpacity": 0.5,
        "shadowRadius": 0.5,
      },
      Object {
        "backgroundColor": "red",
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
        },
        undefined,
      ]
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {},
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          Array [
            Object {
              "color": "#9E9E9E",
              "fontSize": 37.5,
              "fontWeight": "800",
              "textAlign": "center",
            },
            Object {
              "color": "#9E9E9E",
            },
            undefined,
          ],
        ]
      }
    />
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {},
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          Array [
            Object {
              "fontSize": 50,
              "fontWeight": "700",
              "marginBottom": 10,
              "marginTop": 10,
              "textAlign": "center",
            },
            undefined,
          ],
        ]
      }
    />
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {},
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          Array [
            Object {
              "color": "#86939e",
              "fontWeight": "600",
              "marginBottom": 5,
              "marginTop": 5,
              "textAlign": "center",
            },
            undefined,
          ],
        ]
      }
    >
      1 User
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {},
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          Array [
            Object {
              "color": "#86939e",
              "fontWeight": "600",
              "marginBottom": 5,
              "marginTop": 5,
              "textAlign": "center",
            },
            undefined,
          ],
        ]
      }
    >
      Basic Support
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {},
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          Array [
            Object {
              "color": "#86939e",
              "fontWeight": "600",
              "marginBottom": 5,
              "marginTop": 5,
              "textAlign": "center",
            },
            undefined,
          ],
        ]
      }
    >
      All Core Features
    </Text>
    <View
      style={
        Array [
          Object {
            "marginLeft": 15,
            "marginRight": 15,
          },
          undefined,
          undefined,
          undefined,
        ]
      }
    >
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        nativeID={undefined}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            undefined,
          ]
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <View
          pointerEvents="box-only"
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "#9E9E9E",
                "flexDirection": "row",
                "justifyContent": "center",
                "padding": 19,
              },
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              Object {
                "padding": 12,
              },
              undefined,
              undefined,
              undefined,
              Array [
                Object {
                  "marginBottom": 10,
                  "marginTop": 15,
                },
                undefined,
                Object {
                  "backgroundColor": "#9E9E9E",
                },
              ],
              undefined,
              undefined,
            ]
          }
        >
          <Text
            accessible={true}
            allowFontScaling={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "white",
                  "fontSize": 18,
                },
                Array [
                  Object {
                    "marginRight": 10,
                  },
                  undefined,
                ],
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
              ]
            }
          >
            î¤…
          </Text>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Array [
                Object {},
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                Array [
                  Object {
                    "color": "white",
                    "fontSize": 20,
                  },
                  undefined,
                  Object {
                    "fontSize": 17.5,
                  },
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                ],
              ]
            }
          >
            GET STARTED
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;

exports[`PricingCard with no theme applied OldPricingCard and PricingCard returns the same thing if no theme is applied 1`] = `"[2mCompared values have no visual difference.[22m"`;

exports[`PricingCard with no theme applied OldPricingCard renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "white",
        "borderColor": "#e1e8ee",
        "borderWidth": 1,
        "margin": 15,
        "marginBottom": 15,
        "padding": 15,
        "shadowColor": "rgba(0,0,0, .2)",
        "shadowOffset": Object {
          "height": 1,
          "width": 0,
        },
        "shadowOpacity": 0.5,
        "shadowRadius": 0.5,
      },
      undefined,
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
        },
        undefined,
      ]
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {},
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          Array [
            Object {
              "color": "#9E9E9E",
              "fontSize": 37.5,
              "fontWeight": "800",
              "textAlign": "center",
            },
            Object {
              "color": "#9E9E9E",
            },
            undefined,
          ],
        ]
      }
    />
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {},
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          Array [
            Object {
              "fontSize": 50,
              "fontWeight": "700",
              "marginBottom": 10,
              "marginTop": 10,
              "textAlign": "center",
            },
            undefined,
          ],
        ]
      }
    />
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {},
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          Array [
            Object {
              "color": "#86939e",
              "fontWeight": "600",
              "marginBottom": 5,
              "marginTop": 5,
              "textAlign": "center",
            },
            undefined,
          ],
        ]
      }
    >
      1 User
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {},
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          Array [
            Object {
              "color": "#86939e",
              "fontWeight": "600",
              "marginBottom": 5,
              "marginTop": 5,
              "textAlign": "center",
            },
            undefined,
          ],
        ]
      }
    >
      Basic Support
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {},
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          Array [
            Object {
              "color": "#86939e",
              "fontWeight": "600",
              "marginBottom": 5,
              "marginTop": 5,
              "textAlign": "center",
            },
            undefined,
          ],
        ]
      }
    >
      All Core Features
    </Text>
    <View
      style={
        Array [
          Object {
            "marginLeft": 15,
            "marginRight": 15,
          },
          undefined,
          undefined,
          undefined,
        ]
      }
    >
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        nativeID={undefined}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            undefined,
          ]
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <View
          pointerEvents="box-only"
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "#9E9E9E",
                "flexDirection": "row",
                "justifyContent": "center",
                "padding": 19,
              },
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              Object {
                "padding": 12,
              },
              undefined,
              undefined,
              undefined,
              Array [
                Object {
                  "marginBottom": 10,
                  "marginTop": 15,
                },
                undefined,
                Object {
                  "backgroundColor": "#9E9E9E",
                },
              ],
              undefined,
              undefined,
            ]
          }
        >
          <Text
            accessible={true}
            allowFontScaling={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "white",
                  "fontSize": 18,
                },
                Array [
                  Object {
                    "marginRight": 10,
                  },
                  undefined,
                ],
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
              ]
            }
          >
            î¤…
          </Text>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Array [
                Object {},
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                Array [
                  Object {
                    "color": "white",
                    "fontSize": 20,
                  },
                  undefined,
                  Object {
                    "fontSize": 17.5,
                  },
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                ],
              ]
            }
          >
            GET STARTED
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;

exports[`PricingCard with no theme applied PricingCard renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "white",
        "borderColor": "#e1e8ee",
        "borderWidth": 1,
        "margin": 15,
        "marginBottom": 15,
        "padding": 15,
        "shadowColor": "rgba(0,0,0, .2)",
        "shadowOffset": Object {
          "height": 1,
          "width": 0,
        },
        "shadowOpacity": 0.5,
        "shadowRadius": 0.5,
      },
      undefined,
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
        },
        undefined,
      ]
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {},
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          Array [
            Object {
              "color": "#9E9E9E",
              "fontSize": 37.5,
              "fontWeight": "800",
              "textAlign": "center",
            },
            Object {
              "color": "#9E9E9E",
            },
            undefined,
          ],
        ]
      }
    />
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {},
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          Array [
            Object {
              "fontSize": 50,
              "fontWeight": "700",
              "marginBottom": 10,
              "marginTop": 10,
              "textAlign": "center",
            },
            undefined,
          ],
        ]
      }
    />
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {},
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          Array [
            Object {
              "color": "#86939e",
              "fontWeight": "600",
              "marginBottom": 5,
              "marginTop": 5,
              "textAlign": "center",
            },
            undefined,
          ],
        ]
      }
    >
      1 User
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {},
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          Array [
            Object {
              "color": "#86939e",
              "fontWeight": "600",
              "marginBottom": 5,
              "marginTop": 5,
              "textAlign": "center",
            },
            undefined,
          ],
        ]
      }
    >
      Basic Support
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {},
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          Array [
            Object {
              "color": "#86939e",
              "fontWeight": "600",
              "marginBottom": 5,
              "marginTop": 5,
              "textAlign": "center",
            },
            undefined,
          ],
        ]
      }
    >
      All Core Features
    </Text>
    <View
      style={
        Array [
          Object {
            "marginLeft": 15,
            "marginRight": 15,
          },
          undefined,
          undefined,
          undefined,
        ]
      }
    >
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        nativeID={undefined}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            undefined,
          ]
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <View
          pointerEvents="box-only"
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "#9E9E9E",
                "flexDirection": "row",
                "justifyContent": "center",
                "padding": 19,
              },
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              Object {
                "padding": 12,
              },
              undefined,
              undefined,
              undefined,
              Array [
                Object {
                  "marginBottom": 10,
                  "marginTop": 15,
                },
                undefined,
                Object {
                  "backgroundColor": "#9E9E9E",
                },
              ],
              undefined,
              undefined,
            ]
          }
        >
          <Text
            accessible={true}
            allowFontScaling={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "white",
                  "fontSize": 18,
                },
                Array [
                  Object {
                    "marginRight": 10,
                  },
                  undefined,
                ],
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
              ]
            }
          >
            î¤…
          </Text>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Array [
                Object {},
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                Array [
                  Object {
                    "color": "white",
                    "fontSize": 20,
                  },
                  undefined,
                  Object {
                    "fontSize": 17.5,
                  },
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                ],
              ]
            }
          >
            GET STARTED
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;
